gpt_computer_agent/utils/kot_db.py

```
@@ -0,0 +1,11 @@
+try:
+    from .folder import currently_dir, artifacts_dir, media_dir
+except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)